web/packages/teleport/src/Player/DesktopPlayer.tsx

```diff
@@ -73,46 +73,62 @@ export const DesktopPlayer = ({
   const isPlaying = playerStatus === StatusEnum.PLAYING;
   const isComplete = isError || playerStatus === StatusEnum.COMPLETE;
 
-  const t = playerStatus === StatusEnum.COMPLETE ? durationMs : time;
+  const t =
+    playerStatus === StatusEnum.COMPLETE || playerStatus === StatusEnum.ERROR
```

Comment:
Use the existing isComplete variable here.

Reply:
https://github.com/gravitational/teleport/pull/36843/commits/0a76a05884adfbc51e75479aeb7d8fefccca6a99